src/generate.ts

```diff
@@ -240,7 +245,7 @@ export async function generateApi(
     const queryArg: QueryArgDefinitions = {};
     for (const param of parameters) {
       let name = camelCase(param.name);
-      queryArg[name] = {
+      queryArg[param.name] = {
```

Comment:
I'm gonna merge that in but I will add another commit on top of it that will keep the old behaviour for the case that `param.name` matches `/[a-zA-Z][a-zA-Z0-9_]/` to keep the camelcase for the most normal cases.